sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -0,0 +1,317 @@
+        """Create a client from connection string
+
+        :param str connection_string: The connection string of the Service Bus Namespace
+        """
+        endpoint, shared_access_key_name, shared_access_key, _ = parse_conn_str(connection_string)
+        if "//" in endpoint:
+            endpoint = endpoint[endpoint.index("//")+2:]
+        return cls(endpoint, ServiceBusSharedKeyCredential(shared_access_key_name, shared_access_key), **kwargs)
+
+    def _get_queue_object(self, queue_name, **kwargs):
+        # type: (str, Any) -> InternalQueueDescription
+
+        if not queue_name:
+            raise ValueError("queue_name must be a non-empty str")
+
+        with _handle_response_error():
+            et = cast(
+                ElementTree,
+                self._impl.queue.get(queue_name, enrich=False, api_version=constants.API_VERSION, **kwargs)
+            )
+        return _convert_xml_to_object(queue_name, et)
+
+    def _list_queues(self, **kwargs):
+        # type: (Any) -> List[Tuple[str, InternalQueueDescription]]
+
+        skip = kwargs.pop("skip", 0)
+        max_count = kwargs.pop("max_count", 100)
+        with _handle_response_error():
+            et = cast(
+                ElementTree,
+                self._impl.list_entities(
+                    entity_type=constants.ENTITY_TYPE_QUEUES, skip=skip, top=max_count,
+                    api_version=constants.API_VERSION, **kwargs
+                )
+            )
+        entries = et.findall(constants.ENTRY_TAG)
+        queues = []
+        for entry in entries:
+            entity_name = entry.find(constants.TITLE_TAG).text  # type: ignore
+            internal_object = _convert_xml_to_object(
+                entity_name,   # type: ignore
+                cast(Element, entry)
+            )
+            queues.append((entity_name, internal_object))
+        return queues   # type: ignore
+
+    def get_queue(self, queue_name, **kwargs):
+        # type: (str, Any) -> QueueDescription
+        """Get a QueueDescription
+
+        :param str queue_name: The name of the queue
+        """
+        queue_description = QueueDescription._from_internal_entity(  # pylint:disable=protected-access
+            self._get_queue_object(queue_name, **kwargs)
+        )
+        queue_description.queue_name = queue_name
+        return queue_description
+
+    def get_queue_runtime_info(self, queue_name, **kwargs):
+        # type: (str, Any) -> QueueRuntimeInfo
+        """Get the runtime information of a queue
+
+        :param str queue_name: The name of the queue
+        """
+        runtime_info = QueueRuntimeInfo._from_internal_entity(  # pylint:disable=protected-access
+            self._get_queue_object(queue_name, **kwargs)
+        )
+        runtime_info.queue_name = queue_name
+        return runtime_info
+
+    def create_queue(self, queue, **kwargs):
+        # type: (Union[str, QueueDescription], Any) -> QueueDescription
+        """Create a queue
```

Comment:
Nit pick: Please add final period. :)

Reply:
Added period to all docstrings of management client